cocos/ui/UIEditBox/Mac/CCUIEditBoxMac.mm

```diff
@@ -135,6 +135,8 @@ - (NSWindow *)window
 - (void)openKeyboard
 {
     [self.window.contentView addSubview:self.textInput];
+    //FIXME: it will cause a few OpenGL error once, should be fix when glfw version upgraded.
```

Comment:
@rockingdice
You could got a GLFW library from here: https://github.com/cocos2d/cocos2d-x-3rd-party-libs-bin